client/lib/common/src/components/pure/ConditionalHOC.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const Conditional = (condition, ComponentIfTrue, ComponentIfFalse) => class C extends React.Component {
```

Comment:
You could probably make this a real component with props?

~~~ jsx
<If condition={...} when-true={ComponentWhenTrue} else={ElseComponent} />
~~~